src/main/java/codeu/model/data/User.java

```diff
@@ -25,7 +25,7 @@
   private final String passwordHash;
   private final Instant creation;
   private boolean isAdmin;
-  private Set<UUID> conversationSet;
+  private HashSet<UUID> conversationSet;
```

Comment:
why did you change from Set to HashSet? Just trying to understand the logic behind it

Reply:
I was going to use `Set` to be more generic, but it turns out that the `Set` interface isn't serializable,. Since it's not guaranteed to be serializable, it's not guaranteed to have the ability to turn into raw binary for persistent storage. So, I had to use a specific implementation of `Set` that is serializable.